main.py

```diff
@@ -133,6 +149,26 @@ def main():
     activate_ego_upwards_boost = lambda: ego.kinematic.external_forces.append(
         CommonConstants.ROCKET_UPWARD_BOOST_FORCE_SCALAR * Vec2d(0, -1)
     )
+
+    hide_engine_fire_up = lambda: ego_engine_fire.set_color(
+        pygame.Color("orange"),
+        alpha=Opacity.TRANSPARENT,
+    )
+    show_engine_fire_up = lambda: ego_engine_fire.set_color(
+        pygame.Color("orange"),
+        alpha=Opacity.OPAQUE,
+    )
+
+    glue_engine_fire_down_to_ego = lambda: setattr(
+        ego_engine_fire,
+        "pos",
+        Vec2d(
+            ego.rect.midbottom[0],
+            ego.rect.midbottom[1]
+            + 0.5 * CommonConstants.EXHAUST_FIRE_DOWN_DIMENSIONS.y,
+        ),
+    )
```

Comment:
As told via discord, as we probably will glue many objects to another, i would prefer a standardised way so we don't have to do it manually. For this, if we can't find a better solution, i would probably try to make this a functionality of the LandingGameObject.

Reply:
I agree, seeing it as a topic out of scope for this PR.